modules/custom-tuning-example.adoc

```diff
@@ -84,3 +84,36 @@ the `openshift-node` TuneD daemon profile shipped within the default
 Tuned CR to use OpenShift-specific tuning for compute nodes.
 
 // Note the issues with including profiles sharing the same ancestor: see link:https://bugzilla.redhat.com/show_bug.cgi?id=1825882[BZ#1825882]
+
+*Overriding host-level sysctls*
+
+Various kernel parameters can be changed at runtime via `/run/sysctl.d/`, `/etc/sysctl.d/` and `/etc/sysctl.conf` host configuration files.  The {product-title} itself adds several host configuration files which set kernel parameters at runtime. Most notably `net.ipv[4-6].`, `fs.inotify.` and `vm.max_map_count`. These runtime parameters provide very basic functional tuning for the system prior to the kubelet and the Operator start.
+
+The Operator respects these settings and will not override them, unless the `reapply_sysctl` option is set to `false`. This will result in `TuneD` not applying the settings from the host configuration files after it applies its custom profile.
```

Comment:
```suggestion
The Operator does not override these settings unless the `reapply_sysctl` option is set to `false`. Setting this option to `false` results in `TuneD` not applying the settings from the host configuration files after it applies its custom profile.
```

Reply:
I think my suggestion is accurate but I wasn't 100% sure what "this" referred to - setting the option to false, or overriding the settings, or something else.

Reply:
Your suggestion is accurate.  Thank you.  Accepted.